### ISSUE TITLE
Retiring a stake pool instruction is very brief. More detailed step by step needed.

### ISSUE THREAD

**Issue by ItFlyingStart** [input-output-hk/jormungandr#1984]
**Describe the bug**
This is not a bug, but I don't know how to raise my next question: the instruction for "How to retire a stake pool" is very brief (https://github.com/input-output-hk/jormungandr/blob/master/doc/stake_pool/retiring_stake_pool.md).

- Can you provide a more detailed instruction? Can I reuse https://raw.githubusercontent.com/input-output-hk/jormungandr-qa/master/scripts/send-certificate.sh ?
- Where can I find the fee for pool **de**registration?

**Comment by mass-ada-ption** [comment]
Waiting for an answer as well

**Comment by rdlrt** [comment]
@ItFlyingStart  Yes - you can, the fee for deregistration is certificate fee that can be found in your network settings (via jcli network settings get host ...). Check this sample [retireSP.sh](https://github.com/cardano-community/guild-operators/blob/master/scripts/jormu-helper-scripts/retireSP.sh) script for your reference
